scripts/main/view.js

```diff
@@ -1450,31 +1632,37 @@ view.notifications = {
 		view.notifications.content.init();
 	},
 
+	/** @returns {void} */
 	title: function () {
 		lychee.setTitle("Notifications", false);
 	},
 
+	/** @returns {void} */
 	clearContent: function () {
 		lychee.content.html('<div class="settings_view"></div>');
 	},
 
 	content: {
+		/** @returns {void} */
 		init: function () {
 			view.notifications.clearContent();
 
-			$(".settings_view").append('<div class="setting_line"><p>' + `${lychee.locale["USER_EMAIL_INSTRUCTION"]}` + "</p></div>");
-
-			let html = "";
-
-			html +=
-				'<div class="setLogin"><p id="UserUpdate">' +
-				"Enter your email address:" +
-				'<input name="email" class="text" type="text" placeholder="email@example.com" value="' +
-				notifications.json +
-				'">' +
-				'</p><div class="basicModal__buttons">' +
-				'<a id="UserUpdate_button" class="basicModal__button">Save</a>' +
-				"</div></div>";
+			const html = `
+				<div class="setting_line">
+					<p>${lychee.locale["USER_EMAIL_INSTRUCTION"]}</p>
+				</div><div class="setLogin">
+					<p id="UserUpdate">
+						Enter your email address:
+						<input
+							name="email" class="text" type="text"
+							placeholder="email@example.com"
+							value="${notifications.json ? notifications.json.email : ""}"
```

Comment:
```suggestion
							value="${notifications.json && notifications.json.email ? notifications.json.email : ""}"
```

Otherwise, a `null` string will be shown.